src/AccountTile/AccountTile-styled.js

```diff
@@ -31,10 +31,14 @@ const StyledAccountTile = styled.div`
   transition: ${props => props.theme.transition};
   background-color: ${props =>
     props.open ? props.theme.palette.offWhite : props.theme.palette.white};
-  &:hover {
-    background-color: ${props => props.theme.palette.offWhite};
-    cursor: pointer;
-  }
+
+  ${props => props.clickable &&
+    css`
```

Comment:
`css` is undefined here, you'll need to import it from styled components on line 13.